src/rules/disallow-generators-in-describe-functions.js

```diff
@@ -4,15 +4,16 @@
  * Module dependencies.
  */
 
-import assert from 'assert';
+const assert = require('assert');
 
 /**
  * Export `disallowGeneratorsInDescribeFunctions`.
  */
 
-const disallowGeneratorsInDescribeFunctions = () => {};
+module.exports = function() {
+};
 
-disallowGeneratorsInDescribeFunctions.prototype = {
+module.exports.prototype = {
```

Comment:
Not sure if I like this. I would still keep the named `function`, do stuff on its prototype and then export it.